src/main/java/net/testudobank/MvcController.java

```diff
@@ -381,64 +381,28 @@ public String submitDispute(@ModelAttribute("user") User user) {
     int userOverdraftBalanceInPennies = TestudoBankRepository.getCustomerOverdraftBalanceInPennies(jdbcTemplate, userID);
 
     int reversalAmountInPennies = (int) logToReverse.get("Amount");
+    int reversalAmount = reversalAmountInPennies / 100;
```

Comment:
I agree that `reversalAmountInPennies` must be converted back to a dollar amount so that it is a valid value for the `double amountToDeposit` and `double amountToWithdraw` fields in User.java that serve as input for the deposit & withdraw methods.

However, your approach for converting it to a dollar amount is flawed. Firstly, `reversalAmount` should be a `double`, and not an `int`, because you need to be able to account for non-whole dollar amounts. Even though the `double amountToDeposit` field in User.java is a `double`, the Java compiler did not warn you about it because you can technically give an `int` to initialize a `double` field in a Java object.

After making `reversalAmount` a `double`, you should also do `/ 100.0` instead of `/ 100` to force the result of the division to contain a floating point value.

Reply:
TODO: Make all integ tests use non-whole dollar amounts to make sure cases like this are thoroughly tested. (We can do this in a separate PR).

Reply:
Thanks for point that out. I changed `reversalAmount` to a double and changed how I calculated that variable to properly represent a dollar value. I'll keep this in mind when writing integ test.